tricycle_controller/src/tricycle_controller.cpp

```diff
@@ -244,8 +244,8 @@ controller_interface::return_type TricycleController::update(
     auto & realtime_ackermann_command = realtime_ackermann_command_publisher_->msg_;
     // speed in AckermannDrive is defined desired forward speed (m/s) but we use it here as wheel
     // speed (rad/s)
-    realtime_ackermann_command.speed = Ws_write;
-    realtime_ackermann_command.steering_angle = alpha_write;
+    realtime_ackermann_command.speed = static_cast<float>(Ws_write);
```

Comment:
this is ok but I'm tempted to say we should change the messages to use float64 here: https://github.com/ros-drivers/ackermann_msgs/blob/ros2/msg/AckermannDrive.msg

next time...